### ISSUE TITLE
ipython notebooks do not open: "Unreadable Notebook global name 'NBFormatError' is not defined"

### ISSUE THREAD

**Issue by drphilmarshall** [drphilmarshall/Pangloss#39]
I think this is my problem: `ipython --version` returns 2.2, and from some googling it looks like ipython made a breaking change at version 3.0 that causes this error. I am updating my anaconda ipython by doing:

```
conda update conda
conda update ipython ipython-notebook ipython-qtconsole
```

**Comment by drphilmarshall** [comment]
Good, that fixed it - I am now on ipython version 3.2 and apparently using jupyter (not ipython) notebooks, the same as @sweverett.

**Comment by drphilmarshall** [comment]
I also had to 

```
conda update matplotlib
```
